### PR TITLE
Feature/create remove books

### PR DESCRIPTION
**Pull request by mugberto**
# Project: Bookstore
In this branch, we
- implimented adding and removing books functions

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
 const defaultState = [
```
Comment (hemantso) [inline src/redux/books/books.js]: - [ ] As per project requirements, We have to set the initial state to be an empty array of books. Kindly try to set the initial state to an empty array.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @mugberto, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (Davidosky007) [reply src/redux/books/books.js]: * Done 👍

**Comment by Davidosky007** [comment]
## Hi @mugberto 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**